tests/Interface/core/ImageContainerTest.cpp

```diff
@@ -24,6 +24,38 @@
 #include <buddy/Core/Container.h>
 #include <buddy/DIP/ImageContainer.h>
 
+bool compare_flt(float a, float b) {
+  if (a == b)
```

Comment:
No need to have this separate control flow branch here. Please handle it generically with below logic itself.